### ISSUE TITLE
[accessibility] Slider/range controls

### ISSUE THREAD

**Issue by Malvoz** [socib/Leaflet.TimeDimension#211]
The `timecontrol-dateslider` and `timecontrol-speed` sliders are not keyboard accessible and their corresponding output values are not announced by screen readers.

Considering using [`<input type="range">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range) (requires additional styling to reset browser defaults) or the ARIA equivalent: https://www.w3.org/WAI/ARIA/apg/patterns/slider/.

**Comment by Malvoz** [comment]
Other potentially helpful resources:
- [Leaflet Guideline: Test your maps](https://leafletjs.com/SlavaUkraini/examples/accessibility/#test-your-maps)
- [Leaflet Plugin Authoring Guide: Content Accessibility](https://github.com/Leaflet/Leaflet/blob/main/PLUGIN-GUIDE.md#content-accessibility)

**Comment by r1m** [comment]
Time slider also includes range limiting knobs : https://apps.socib.es/Leaflet.TimeDimension/examples/example3.html
![image](https://user-images.githubusercontent.com/137285/164031265-592b1f35-6117-4a52-bfb5-0c2c884c0521.png)

**Comment by Malvoz** [comment]
@r1m There's also a [Slider (Multi-Thumb)](https://www.w3.org/TR/wai-aria-practices-1.1/#slidertwothumb) ARIA design pattern ([example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/slider/multithumb-slider.html)).
